### DISCUSSION TITLE
Multipoint Optimisation

### DISCUSSION THREAD

**Discussion by Otherside5** [mdolab/dafoam#365] [General]
Hello everyone,

I have a general question regarding multipoint optimisation with DAFoam.

I have a hydro pump/turbine and want to optimise the turbine wheel on several operating points. The idea would be too loop over different rotation speeds and boundary conditions to achieve that. In DAFoam you can set design variable type as "BC" and loop over them. My problem is that I want to use data that I have from a different simulation as boundary conditions (csv files, dat files) with a non uniform list or a fixed profile and not a simple fixed value BC.

Is it possible to set such a case up with DAFoam? And if it is how can I load the csv files during the optimisation?

Thanks in advance
Alex

**Comment by friedenhe** [comment]
This is doable by using the multicase feature. We just created a new tutorial for it: https://github.com/DAFoam/tutorials/tree/main/NACA0012_Airfoil/multicase. In this case, we create two separate cases, one uses the SA model and the other uses the SST model. Their CFD and adjoint run independently but they share the same airfoil shape. The objective is the averaged drag between the SA and SST case. You can apply a similar approach to your case.

**Comment by Otherside5** [comment]
That is exactly what I need.

Thank you very much!
